File: 学习与算法/openmp/1. openmp/2. constructs/lab_openmp/lab_openmp/hello.c
#include <stdio.h>
#include <omp.h>
int main(int argc, char *argv[]){

   printf("Hello from primary thread.\n");

#pragma omp parallel
   {
      printf("Team member %d reporting from team of %d\n",
             omp_get_thread_num(),omp_get_num_threads() );
   }

   printf("Primary thread finished, goodbye.\n");

}

File: 学习与算法/openmp/1. openmp/2. constructs/lab_openmp/lab_openmp/hello.f90
program hello

   use omp_lib

   print *, "Hello from primary thread."

!$omp parallel

   print *, "Team member ",omp_get_thread_num(), &
  &         "reporting from team of ",omp_get_num_threads(),"."

!$omp end parallel

   print *, "Primary thread finished, goodbye."

end program hello
